src/main/kotlin/com/figure/gradle/semver/internal/tasks/CurrentSemverTask.kt

```diff
@@ -13,8 +13,6 @@ import org.gradle.api.provider.Property
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
 import org.gradle.work.DisableCachingByDefault
-
-@DisableCachingByDefault(because = "Not worth caching")
 internal abstract class CurrentSemverTask : DefaultTask() {
```

Comment:
I could be wrong, but if you want to denote this as a specifically cacheable task, I think we should annotate it with `@CacheableTask`. It at least helps us as devs identify if it _should be_ cacheable or not. I don't think it's just a marker annotation, but I could be wrong.

Reply:
Added